docs/FAQ.md

```diff
@@ -71,11 +71,14 @@ Since we expect the above to be straightforward for most users and it allows the
 **NOTE** Releases of 0.3.2 onwards should not have this issue, or require this solution.
 
 If you see `?` in place of emojis or Zulip Terminal gives a `UnicodeError` / `CanvasError`, you haven't enabled utf-8
-encoding in your terminal. To enable it by default, add this to the end of you `~/.bashrc`:
+encoding in your terminal. To enable it by default, add this to the end of your `~/.bashrc`:
```

Comment:
Should we specify the shell in a more general way? For e.g. Mac uses `ZSH` if I am correct, so they won't have a `bashrc` but a `zshrc`. I personally moved on to ZSH quite recently too, and although I still have a `bashrc` adding the below line there probably won't work for me(?)

Reply:
Updated, does this seem more general?